saleor/order/migrations/tasks/saleor3_19.py

```diff
@@ -1,21 +1,25 @@
 from decimal import Decimal
 
 from django.db import transaction
-from django.db.models import Exists, F, OuterRef, Q
+from django.db.models import Exists, F, OuterRef, Q, Sum
 from django.forms.models import model_to_dict
 
 from ....account.models import Address
 from ....celeryconf import app
 from ....discount.models import OrderDiscount, Voucher
 from ....warehouse.models import Warehouse
 from ...models import Order, OrderLine
+from ...utils import updates_amounts_for_order
 
 # The batch of size 250 takes ~0.5 second and consumes ~20MB memory at peak
 ADDRESS_UPDATE_BATCH_SIZE = 250
 
 # The batch of size 250 takes ~0.2 second and consumes ~20MB memory at peak
 ORDER_SET_SHIPPING_PRICE_BATCH_SIZE = 250
 
+# The batch of size 50 takes ~0.12 second and consumes 3.8MB memory
+ORDER_SET_SUBTOTAL_PRICE_FOR_ORDER_FROM_BULK = 50
```

Comment:
Maybe we good to increase the batch size then?

Reply:
Increased to 100